components/dashboard/src/index.css

```diff
@@ -35,23 +35,29 @@
 
 @layer components {
     button {
-        @apply cursor-pointer px-4 py-2 my-auto bg-green-600 hover:bg-green-700 border-2 border-green-800 text-gray-100 text-sm font-medium rounded-md focus:outline-none focus:ring focus:border-blue-300;
+        @apply cursor-pointer px-4 py-2 my-auto bg-gitpod-kumquat-dark hover:bg-gitpod-kumquat-darker text-white text-sm font-medium rounded-md focus:outline-none focus:ring;
```

Comment:
**issue**: Strong vote for not changing the primary color to kumquat although this looks yummy! Orange (~kumquat) color usually indicates a warning or something is pending. But we could try this and revert later. 😬

Reply:
I understand. I feel it makes the branding stronger and with that, it doesn't feel so much signaling but fruity.